src/bundle/SystemInfo/Value/IbexaSystemInfo.php

```diff
@@ -49,41 +49,67 @@ class IbexaSystemInfo extends ValueObject implements SystemInfo
 
     /**
      * @var bool
+     *
+     * @uses $endOfMaintenanceDate
      */
     public $isEndOfMaintenance = true;
 
     /**
-     * @var \DateTime
+     * @var \DateTime EOM for the given release, if you have an Ibexa DXP / Enterpise susbscription.
+     *
+     * @see https://support.ibexa.co/Public/Service-Life
      */
     public $endOfMaintenanceDate;
 
     /**
      * @var bool
+     *
+     * @uses $endOfLifeDate
      */
     public $isEndOfLife = true;
 
     /**
-     * @var \DateTime
+     * @var \DateTime EOL for the given release, if you have an Ibexa DXP susbscription.
+     *
+     * @see https://support.ibexa.co/Public/Service-Life
      */
     public $endOfLifeDate;
 
+    /**
+     * @var bool If install should have subscription data (due to enterpise packages), but it's not present.
```

Comment:
I don't think the last part is true actually. Reading through the code that sets it, it just changes if at least one enterprise package is installed.

```suggestion
     * @var bool If installation should have subscription data (due to enterpise packages).
```

Reply:
The last part is only true once the other PR is some form is accepted.

@alongosz I would suggest dropping this property from this PR, it only makes sense in context of the other.

Reply:
It was rather a placeholder for the future, but dropped since it's confusing (243fb729)